design-docs/008-command-line-interface.md

```diff
@@ -0,0 +1,143 @@
+# Icechunk Command Line Interface
+
+This document outlines the design of the Icechunk command line interface.
+
+## Functionality
+
+Here is a list of tasks a user might want to do with Icechunk:
+
+- List my repositories
```

Comment:
What should this list? Probably all repositories in the repositories config? 

It would also be great to be able to point at a location and auto-discover repos.